moco/src/main/java/com/cognifide/aem/stubs/moco/MocoStubs.java

```diff
@@ -83,11 +98,25 @@ protected void deactivate() {
 
   private void start() {
     LOG.info("Starting AEM Stubs Moco Server");
+
     if (config.logging()) {
-      server = httpServer(config.port(), ApiUtils.log(LOG::info));
+      server = (ActualHttpServer) httpServer(config.port(), ApiUtils.log(LOG::info));
     } else {
-      server = httpServer(config.port());
+      server = (ActualHttpServer) httpServer(config.port());
     }
+
+    resolverAccessor.consume(resolver -> {
```

Comment:
There are few places in code which reads files from JCR. Maybe its worth to bring it to a common? I mean class ```JcrFileReader```and ```Repository```

Reply:
yep, refactor it in the future. no time for it right now